src/main/java/net/neoforged/neoforge/common/extensions/IItemStackExtension.java

```diff
@@ -395,6 +397,13 @@ default <T> T getCapability(ItemCapability<T, @Nullable Void> capability) {
         return capability.getCapability(self(), null);
     }
 
+    /**
+     * @return A {@link ResourceStack} with the current count. This resource stack is fully immutable in comparison.
```

Comment:
```suggestion
     * {@return a {@linkplain ResourceStack} with the current count} This resource stack is immutable.
```
Use an inline return tag and remove the "in comparison" part, the mention that the resource stack is immutable is enough.

Reply:
This doesn't fully work when using read modes. While I'm ok with the documentation changes, the tagging seems to be incorrect.
Your result:
![image](https://github.com/user-attachments/assets/0862425f-4615-4f5d-85c7-e403141778ae)

Instead I think we can do this to compromise between the two
![image](https://github.com/user-attachments/assets/45607425-2f88-43a8-b31a-983c36287174)

Reply:
The tagging is not incorrect, if you check the generated javadoc website you will see it displays correctly, as is intended for inline return tags to do. What you're observing is an intellij bug that was reported. Regardless, we should not write our documentation based on IDE bugs which ought to be fixed at any time.